django_cloud_deploy/skeleton/source_generator.py

```diff
@@ -337,95 +442,150 @@ def generate_from_existing(self,
 class _DockerfileGenerator(_Jinja2FileGenerator):
     """Generate Dockerfile to build image for the Django project."""
 
-    def generate(self, project_name: str, destination: str):
+    _FILES = ('Dockerfile', '.dockerignore')
+
+    @staticmethod
+    def generated(project_dir: str) -> bool:
+        files_list = os.listdir(project_dir)
+        for file_name in _DockerfileGenerator._FILES:
+            if file_name not in files_list:
+                return False
+        return True
+
+    def generate(self, project_name: str, project_dir: str):
+        if not self.generated(project_dir):
+            self._generate_new(project_name, project_dir)
+
+    def _generate_new(self, project_name: str, project_dir: str):
         """Generate Dockerfile and .dockerignore.
 
         Args:
             project_name: The name of your Django project.
-            destination: The destination directory path to put Dockerfile.
+            project_dir: The destination directory path to put Dockerfile.
         """
         file_names = ('Dockerfile', '.dockerignore')
         options = {'project_name': project_name}
         for file_name in file_names:
             template_path = os.path.join(self._get_template_folder_path(),
                                          file_name)
-            output_path = os.path.join(destination, file_name)
+            output_path = os.path.join(project_dir, file_name)
             self._render_file(template_path, output_path, options)
 
+
 class _AppEngineFileGenerator(_Jinja2FileGenerator):
     """Generate App Engine Files for the Django project."""
 
+    _FILES = ('.gcloudignore', 'app.yaml')
+
+    @staticmethod
+    def generated(project_dir: str) -> bool:
+        files_list = os.listdir(project_dir)
+        for file_name in _AppEngineFileGenerator._FILES:
+            if file_name not in files_list:
+                return False
+        return True
+
     # TODO: remove database_password
-    def generate(self, project_name: str, destination: str,
+    def generate(self, project_name: str, project_dir: str,
                  database_password: str):
         """Generate Dockerfile and .dockerignore.
```

Comment:
Meh copy paste error from my PR. If you can fix it ill appreciate it, else I'll do it in my next PR. :)

Reply:
Should be App.yaml + .dockerignore.

Reply:
Done.